participant/app/routes/$.tsx

```diff
@@ -0,0 +1,15 @@
+import { redirect } from "@remix-run/server-runtime";
+import type { LoaderFunction } from "@remix-run/server-runtime";
+import { validRoute } from "~/utils/redirect";
+
+export const loader: LoaderFunction = async ({ request, params }) => {
+  const url = new URL(request.url);
+  if (url.pathname.includes("favicon.ico")) {
+    return null;
```

Comment:
I don't know why we're getting requests for `/gallatin/recertify/favicon.ico` -- but because we are, those requests are hitting the splat route. 
Requests for `/favicon.ico` work perfectly fine.